### DISCUSSION TITLE
Feature request - Quick reply buttons support

### DISCUSSION THREAD

**Discussion by ftore** [flyerhq/flutter_chat_ui#684] [Q&A] [answered]
Since mobile integration with LLMs - ChatGTP, Anthropic and etc - is getting more and more popular, a lot of projects will be creating chatbots. It would be great, we flutter_chat_ui supports custom widget support along with standard chat bubble. We could use custom widget to present quick reply buttons, or action buttons such as accept / decline offer, or yes or no buttons to bot's questions and etc. Currently, the only option we have is bubbleBuilder - [https://github.com/flyerhq/flutter_chat_ui/issues/294](url) or [https://github.com/flyerhq/flutter_chat_ui/issues/301](url). But it is too difficult to implement.

**Comment by demchenkoalex** [comment, marked as answer]
`bubbleBuilder` is a legacy. While I don't have that yet in `v2` it will be probably easy to add. In the backlog I have:
- Allow top bottom leading trailing widgets for bubble https://github.com/flyerhq/flutter_chat_ui/discussions/653
- Quick responses above the input https://github.com/flyerhq/flutter_chat_ui/discussions/188

I think this covers all possible actions
